superset/assets/src/visualizations/line_multi.js

```diff
@@ -26,27 +25,25 @@ export default function lineMulti(slice, payload) {
     const fdCopy = {
       ...subslice.form_data,
       filters,
-      since: fd.since,
-      until: fd.until,
+      time_range: fd.time_range,
```

Comment:
The key change is this line. vs line 29-30 from previous. 
The other changes in this file are refactor.